src/content/guides/third-party-application-payments.mdx

```diff
@@ -3,7 +3,7 @@ title: Paying a Payment Request
 description: Integrate your app with Centrapay to pay payment requests on merchant terminals and POS
 nav:
   path: Reference/App Integrations
-  order: 1
+  order: 2
```

Comment:
This order affects the order of the page in its respective sub header. This page is sitting in the "App Integrations" header, and no page has been added to that header as part of this PR. You shouldn't need to change this.